### ISSUE TITLE
Benefit Letters - Heading order violation - Progress Stepper

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108004]
## Issue
The Progress stepper is an H4. It comes after the H1, and before an H2... 

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: Pages with the progress stepper ex) https://staging.va.gov/records/download-va-letters/letters/confirm-address

<img width="1483" alt="Image" src="https://github.com/user-attachments/assets/1b74e0ce-5556-4faf-b682-650a1b88e7b9" />

**Comment by bacitracin** [comment]
Covered in 108007, closing as dupe
